GameBoard/Models/GameEventInviteForm/GameEventInviteFormViewModel.cs

```diff
@@ -8,11 +10,23 @@ public class GameEventInviteFormViewModel
 
         public string UserName { get; }
 
-        public GameEventInviteFormViewModel(int gameEventId, string gameEventName, string userName)
+        public int GroupId { get; }
+
+        public GroupViewModel Group { get; }
+
+        public GameEventInviteFormViewModel(int gameEventId, string gameEventName, string userName, int groupId)
         {
             GameEventId = gameEventId;
             GameEventName = gameEventName;
             UserName = userName;
+            GroupId = groupId;
+        }
+
+        public GameEventInviteFormViewModel(int gameEventId, string gameEventName, GroupViewModel group)
+        {
+            GameEventId = gameEventId;
+            GameEventName = gameEventName;
+            Group = group;
         }
```

Comment:
What's going on here? Why do we have two different ctors that set up two different sets of member properties? What's the responsibility of this class anyway?